enhance-dictionary.pl

```diff
@@ -260,7 +260,7 @@ sub load_edict {
   $edict{'__TYPE'} = 'edict2';
   $edict{'__FILE'} = $edict;
   $edict{'__USED'} = 0;
-  open (my $wf, '<:encoding(utf8)', $edict) or die "Cannot open $edict: $?";
+  open (my $wf, '<:encoding(euc-jp)', $edict) or die "Cannot open $edict: $?";
```

Comment:
Are edic files always in euc-jp encoding?

Reply:
Yes, according to the Format section on http://www.edrdg.org, edict files are always in euc-jp encoding (or encapsulation):
![screen shot 2019-02-07 at 22 07 51](https://user-images.githubusercontent.com/28924027/52443158-4ac94700-2b25-11e9-8264-f474215ff8e9.png)